### PR TITLE
Set up travis

### PR DESCRIPTION
**Pull request by dkhofer**

### CODE REVIEW

File: .rspec [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--color
+--format progress
```
Comment (tmorris-fiksu) [inline .rspec]: Especially for a small project like this, I'd choose "--format documentation".  (shrug)

**Comment by tmorris-fiksu** [comment]
No yet hooked up to Travis?

File: activerecord-redshift-adapter.gemspec [hunk 2]
```diff
@@ -17,8 +17,8 @@ Gem::Specification.new do |s|
...
  s.homepage    = 'http://github.com/fiksu/activerecord-redshift-adapter'
  s.add_dependency "pg"
- s.add_dependency "activerecord", '>= 3.0.0'
- s.add_dependency "activesupport", '>= 3.0.0'
- s.add_dependency "arel", '>= 3.0.0'
- s.add_development_dependency 'rspec'
+ s.add_dependency "activerecord", '~> 3.0'
+ s.add_dependency "activesupport", '~> 3.0'
+ s.add_dependency "arel", '~> 3.0'
+ s.add_development_dependency 'rspec', '~> 3.0'
```
Comment (tmorris-fiksu) [inline activerecord-redshift-adapter.gemspec]: What's the status of Rails 4 support for this gem?

File: spec/active_record/connection_adapters/redshift_adapter_spec.rb [hunk 3]
```diff
@@ -9,6 +9,7 @@
     CREATE TABLE public.test2 ( "id" INTEGER, "name" VARCHAR );
     INSERT INTO public.test VALUES (1, 'first');
     INSERT INTO public.test VALUES (2, 'second');
+    CREATE SCHEMA test;
```
Comment (tmorris-fiksu) [inline spec/active_record/connection_adapters/redshift_adapter_spec.rb]: I recall that the `create schema` and then the `create table` that depends on it have to be sent in a different string.  Redshift parses all the statements first, apparently.  I ran this against Redshift w/ failure:
```
  1) ActiveRecord::ConnectionAdapters::RedshiftAdapter#initialize opens a connection
     Failure/Error: @connection.query <<-sql
     ActiveRecord::StatementInvalid:
       PG::InvalidSchemaName: ERROR:  schema "test" does not exist
       :     CREATE TABLE public.test ( "id" INTEGER NULL, "name" VARCHAR(80) NULL );
           CREATE TABLE public.test2 ( "id" INTEGER, "name" VARCHAR );
           INSERT INTO public.test VALUES (1, 'first');
           INSERT INTO public.test VALUES (2, 'second');
           CREATE SCHEMA test;
           CREATE TABLE test.test ( "id" INTEGER NOT NULL, "is" BOOL NOT NULL );
           CREATE TABLE test.test2 ( "id" INTEGER, "is" BOOL );
```

File: activerecord-redshift-adapter.gemspec (hunk 2 above)
Comment (dkhofer) [inline activerecord-redshift-adapter.gemspec]: So I believe the current code supports up to rails 4.1 but not 4.2, which is what this PR addresses: https://github.com/fiksu/activerecord-redshift-adapter/pull/22

I believe the only people affected by the "~> 3.0" change are people pulling directly from HEAD on this repo.  If they care, they can specify v0.9.11 to get a version that can "work" with rails 4.0 or 4.1.

Longer term, my plan is to tag a new version for rails 3 only (which includes this branch's changes), then merge a change that specifies rails 4.0 or 4.1 as the dependency and tag a new version for that, then deal with 4.2 compatibility.  Sound good?

File: .rspec (hunk 1 above)
Comment (dkhofer) [inline .rspec]: Changed.

File: spec/active_record/connection_adapters/redshift_adapter_spec.rb (hunk 3 above)
Comment (dkhofer) [inline spec/active_record/connection_adapters/redshift_adapter_spec.rb]: Good point.  Changed.

**Comment by dkhofer** [comment]
OK, travis is enabled for this repo now.

**Comment by tmorris-fiksu** [comment]
LGTM.

**Pull request merged**